fe/src/project/AspNetCore/areas/DownloadCenter/components/DownloadCenterCatalog/DownloadCenterCatalog.scss

```diff
@@ -0,0 +1,17 @@
+.dc-catalog {
+    &__container {
+        margin-bottom: 3rem;
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+
+    &__list-item {
```

Comment:
**suggestion (non-blocking):** wouldn't a more meaningful name like &__subcategory-link be better here?